README.md

```diff
@@ -14,83 +14,260 @@ A curated list of anime streaming/downloading websites.
 * I am **NOT** promoting the use of free anime websites by any means.
 
 
-## Contents
+# Contents
+* [Anime](#Anime)
+ * [Streaming](#Streaming)
```

Comment:
Streaming is not coming as a subsection of anime, need to fix indentation I think

Reply:
try now